### PR TITLE
Update the ResNet model for TensorFlow.

### PR DESCRIPTION
**Pull request by qingqing01**
Fix https://github.com/dzhwinter/benchmark/issues/28

### CODE REVIEW

File: tensorflow/resnet.py [hunk 1]
```diff
@@ -443,38 +397,73 @@ def run_benchmark(args, data_format='channels_last', device='/cpu:0'):
...
+                    test_accs.append(
+                        accuracy.eval(feed_dict={
+                            images: test_images,
+                            labels: test_labels,
+                            is_training: False
+                        }))
+                print(
+                    "Pass = %d, Train performance = %f imgs/s, Test accuracy = %f\n"
+                    %
+                    (pass_id, num_samples / train_elapsed, np.mean(test_accs)))
```
Comment (chengduoZH) [inline tensorflow/resnet.py]: test这一块写成一个函数是不是更合适些， `for pass_id in range(args.pass_num):`这个循环体显得很长

File: tensorflow/resnet.py [hunk 2]
```diff
@@ -117,6 +121,7 @@ def conv2d_fixed_padding(inputs, filters, kernel_size, strides, data_format):
     """Strided 2-D convolution with explicit padding."""
     # The padding is consistent and is based only on `kernel_size`, not on the
     # dimensions of `inputs` (as opposed to using `tf.layers.conv2d` alone).
+    # This is consistent with PaddlePaddle.
```
[the remark this answers is not in the document]
Comment (qingqing01) [reply tensorflow/resnet.py]: Done.

File: tensorflow/resnet.py (hunk 1 above)
Comment (qingqing01) [reply tensorflow/resnet.py]: Done.

**Comment by chengduoZH** [comment]
LGTM++

**Pull request merged**